canonical-websites/www.ubuntu.com
# Removed nav-section dropdowns for items without children

## Done

Dropdowns were being created for nav sections without children on small screens. Now the nav sections do not create an accordion for nav section heading without children. 
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Minimise home screen and look at footer the links after the sector section should not be in accordion
format.


## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4140
## Screenshots

![image](https://user-images.githubusercontent.com/43535482/47607491-92b1db80-da18-11e8-99ef-3d9b148a4f10.png)
